templates/patroni.yml.j2

```diff
@@ -64,6 +64,11 @@ bootstrap:
         {%- endif %}
         archive_mode: on
         wal_level: logical
+        {%- if pg_parameters %}
+        {%- for key, value in pg_parameters.items() %}
+        {{key}}: {{value}}
+        {%- endfor -%}
+        {% endif %}
```

Comment:
I think if we pass an empty dict as a default `pg_parameters` we can remove the outer if, but that can be changed later.

Reply:
I'm think I've tested this, but will recheck

Reply:
> I'm think I've tested this, but will recheck

Not worth rerunning CI over it.